client/scss/components/_header_block.scss

```diff
@@ -0,0 +1,107 @@
+//* @define header-block;
+
+.header-block {
+  &.header-block--a {
+    @include respond-to('large') {
+      position: relative;
+      padding: 1.8rem 0 1.5rem span-columns('large', 1.5);
+
+      &:before {
+        content: '';
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        width: span-columns('large', 0.5);
+        background: color('keyline-grey');
+      }
+    }
+  }
+
+  &.header-block--b,
+  &.header-block--c:not(.header-block--xlarge) {
+    padding-bottom: 1.2rem;
+    border-bottom: 1px solid color('keyline-grey');
+  }
+}
+
+.header-block__title {
+  margin-top: 0;
+  margin-left: -0.06em;
+
+  .header-block--a & {
+    @include font('WB3');
+    margin-bottom: 0.25em;
+  }
+
+  .header-block--b & {
+    @include font('HNM1');
+  }
+
+  .header-block--c.header-block--small & {
```

Comment:
Composing classes like this is defeating of a lot of the BEM principles. It means the SASS is composed, where as we want to be able to do this in the HTML.

Adding more classes is more BEMy.

We could have:
`.header-block--c { @include font('WB6'); }`
`.header-block--small { margin-bottom: 0.4em; }`
```

Then I can add those classes wherever I want.